packages/perseus/src/types.ts

```diff
@@ -542,6 +543,11 @@ export type WidgetScorerFunction = (
     locale?: string,
 ) => PerseusScore;
 
+/**
+ * A union type of all the functions to get the public options for each widget.
+ */
+export type PublicWidgetOptionsFunctions = typeof getCategorizerPublicOptions;
```

Comment:
This currently only has categorizer's public options function, but as I create more, I'll keep adding to this. I wanted to do some kind of generic here, but that spiraled a bit, so staying with what seemed to work,